### PR TITLE
code.talks 2016 retrospective

### PR DESCRIPTION
**Pull request by jensfischerhh**

### CODE REVIEW

File: _posts/2016/2016-10-04-code-talks-2016-retrospective.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+
+{% image blog/blog-codetalks2016-talk.jpg  %}
+
+
+## Highlights
+
+### The Return of Sanity? (Die Rückkehr der Vernunft?)
+
+The conference started with a real highlight on the main stage in K6 - [Lars Jankowfsky](https://twitter.com/dodgeris) and [Johann-Peter Hartmann](https://twitter.com/Johannhartmann) gave a talked about *"The Return of Sanity?" ("Die Rückkehr der Vernunft?")*. Their talk was a discussion between the two about which technology stack to use for their imaginary starup *canab.io*. This was a very entertaining sequence of bashing hot programming languages.
```
Comment (ubep) [inline _posts/2016/2016-10-04-code-talks-2016-retrospective.md]: typo: startup

File: _posts/2016/2016-10-04-code-talks-2016-retrospective.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The sponsor often has different goals. The choosen technology should scale and it should be possible to find skilled people on the market. Also they do not want to be dependent on a certain technology and it should leverage fast time to market.
+
+<blockquote class="twitter-tweet" data-conversation="none" data-cards="hidden" data-lang="de"><p lang="de" dir="ltr"><a href="https://twitter.com/zaddo">@zaddo</a> aber jetzt wird es ernst <a href="https://twitter.com/hashtag/codetalkshh?src=hash">#codetalkshh</a> <a href="https://t.co/cBf35RfAwj">pic.twitter.com/cBf35RfAwj</a></p>&mdash; Mathias D (@zaddo) <a href="https://twitter.com/zaddo/status/781412298263060480">29. September 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+So their appeal was to choose technology more scientifically and not based on hype - they asked everybody to decide as if you have *"skin in the game"*. Which technology would you choose if the south italian investor captured your family and will just release them if the project is develivered in time.
```
Comment (ubep) [inline _posts/2016/2016-10-04-code-talks-2016-retrospective.md]: typo: delivered

**Comment by jensfischerhh** [comment]
@ubep @mduesterhoeft @roseritter @renatoavt please give some final feedback before merging this post on Friday

File: _posts/2016/2016-10-07-code-talks-2016-retrospective.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+Actually the talk was rather a play than a talk.
+It was a discussion between the two about which technology stack to use for their imaginary startup *canab.io*.
+They started going through a lot of hot technologies only to bash them all.
+So pure entertainment on the main stage.
+
+<blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">Pures Entertainment <a href="https://twitter.com/hashtag/codetalkshh?src=hash">#codetalkshh</a> 🚀💯 <a href="https://t.co/V86LhW3qdU">pic.twitter.com/V86LhW3qdU</a></p>&mdash; Mathias D (@zaddo) <a href="https://twitter.com/zaddo/status/781407595202539521">29. September 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The point Lars and Johann tried to make was actually a serious one.
+They suggested that developers often choose technologies because they are interested in learning them and not by experience and profound knowledge.
```
Comment (jensfischerhh) [inline _posts/2016/2016-10-07-code-talks-2016-retrospective.md]: They argued that...

**Comment by renatoavt** [comment]
Besides the last comment, looks good!

**Pull request merged**